.pipelines/templates/ios_shared_lib_build.yml

```diff
@@ -0,0 +1,71 @@
+# packaging pipeline for onnxruntime-extensions Android AAR package
+parameters:
+- name: Platform
+  displayName: ios Platform
+  type: string
+- name: ios_arch
+  displayName: ios_arch
+  type: string
+- name: IsReleaseBuild
+  displayName: "Is this is a release build?"
+  type: boolean
+- name: UseDistributedBuild
+  displayName: UseDistributedBuild or not 
+  type: boolean
+
+  
+jobs:
+  - job: IOS_C_API_Packaging_${{ parameters.Platform }}_${{ parameters.ios_arch }}
+    condition: and(succeeded(), eq(true, ${{ parameters.UseDistributedBuild}}) )
+    pool:
+      vmImage: "macOS-12"
+    timeoutInMinutes: 120
+    variables:
+      buildConfig: Release
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+          addToPath: true
+          architecture: "x64"
+
+      # iOS xcframework build doesn't work with CMake 3.25.1, pin to 3.25.0
+      - script: |
+          python -m pip install cmake==3.25.0
+        displayName: "Install CMake 3.25.0"
+
+      - template: set-package-version-variable-step.yml
+        parameters:
+          IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+          PackageVersionVariableName: ORT_EXTENSIONS_POD_VERSION
+
+      - script: |
+          python ./tools/gen_selectedops.py ./tools/ios/package_ops.config
```

Comment:
same comment here regarding the nuget build needing to include the same ops across all platforms